src/main/resources/music/settings/host/values/strings.xml

```diff
@@ -148,31 +149,31 @@ Info:
     <string name="revanced_custom_filter_strings_summary">List of component path builder strings to filter, separated by new lines.</string>
     <string name="revanced_custom_filter_toast_invalid_syntax">Invalid custom filter: %s.</string>
     <string name="revanced_hide_button_shelf_title">Hide button shelf</string>
-    <string name="revanced_hide_button_shelf_summary">Hides button shelf in feed.</string>
+    <string name="revanced_hide_button_shelf_summary">Hides the button shelf in the feed.</string>
     <string name="revanced_hide_carousel_shelf_title">Hide carousel shelf</string>
-    <string name="revanced_hide_carousel_shelf_summary">Hides carousel shelf in feed.</string>
+    <string name="revanced_hide_carousel_shelf_summary">Hides the carousel shelf in the feed.</string>
     <string name="revanced_hide_playlist_card_shelf_title">Hide playlist card shelf</string>
```

Comment:
Uhh, idk if these are proper nouns

(`button shelf`, `carousel shelf`, and `playlist card shelf`)

Reply:
This is an alternate word I chose because I couldn't find a proper formal name for it (it's not a proper noun)